packages/hebao_v1/contracts/stores/NonceStore.sol

```diff
@@ -31,6 +31,7 @@ contract NonceStore is DataStore
 
     function verifyAndUpdate(address wallet, uint nonce)
         public
+        onlyWalletModule(wallet)
```

Comment:
do we have the same issue in other Stores?

Reply:
I think the other stores are fine (and they are still pretty similar to the previous versions I think).